enclave/core/optee/globals.c

```diff
@@ -18,14 +18,15 @@ extern const size_t ta_heap_size;
 **==============================================================================
 */
 
-const void* __oe_get_enclave_base()
+const void* __oe_get_enclave_start_address()
```

Comment:
Need to also have the implementation of `__oe_get_enclave_base` for OPTEE given that we're keeping it for SGX